src/components/NavMain/NavMain.tsx

```diff
@@ -26,10 +24,10 @@ const NavMain = () => {
 
   return (
     <nav role="navigation">
-      <ul className={styles.navigation}>
+      <ul className="flex justify-center h-48">
```

Comment:
I replaced the CSS from the module with the Tailwind equivalent as a starting point.